integration/fuse/src/main/java/alluxio/fuse/AlluxioFuse.java

```diff
@@ -41,15 +43,47 @@
 public final class AlluxioFuse {
   private static final Logger LOG = LoggerFactory.getLogger(AlluxioFuse.class);
 
+  private static final Option MOUNT_POINT_OPTION = Option.builder("m")
+      .hasArg()
+      .required(true)
+      .longOpt("mount-point")
+      .desc("Desired local mount point for alluxio-fuse.")
+      .build();
+
+  private static final Option ALLUXIO_ROOT_OPTION = Option.builder("r")
+      .hasArg()
+      .required(true)
+      .longOpt("alluxio-root")
+      .desc("Path within alluxio that will be used as the root of the FUSE mount "
+          + "(e.g., /users/foo; defaults to /)")
+      .build();
+
+  private static final Option HELP_OPTION = Option.builder("h")
+      .required(false)
+      .desc("Print this HELP_OPTION")
+      .build();
+
+  private static final Option FUSE_MOUNT_OPTION = Option.builder("o")
+      .valueSeparator(',')
+      .required(false)
+      .hasArgs()
+      .desc("FUSE mount options")
+      .build();
+
+  private static final Options OPTIONS = new Options()
+      .addOption(MOUNT_POINT_OPTION)
+      .addOption(ALLUXIO_ROOT_OPTION)
+      .addOption(HELP_OPTION)
+      .addOption(FUSE_MOUNT_OPTION);
```

Comment:
NOTE, this is some cleanup work I planned to do a long time ago. unrelated to @iluoeli 's change to switch to JNI